### PR TITLE
Msms frag ratio

### PR DESCRIPTION
**Pull request by bkieft-usa**
First attempt at implementing a filter for C18 analysis that uses the ratio of matched fragment ions to reference ions in MSMS data.

### CODE REVIEW

File: metatlas/tools/atlas_prefilter.py
Comment (tharwood3) [inline metatlas/tools/atlas_prefilter.py:125]: We should remove this diagnostic csv saving line before merging into main.

**Review by tharwood3 — COMMENTED** [review]
Looks good to me! My only note is to remove the diagnostic csv saving line. As far as the CI failure goes, I'm not really sure why that error would have appeared -- let's see what happens after making another commit and running the CI checks again.

File: metatlas/tools/atlas_prefilter.py
Comment (bkieft-usa) [reply metatlas/tools/atlas_prefilter.py:125]: Ha, thanks for catching that. Looking at the error this afternoon.

Comment (bkieft-usa) [reply metatlas/tools/atlas_prefilter.py:125]: Also, I'm going to keep the other ratio just for reference (reference_data_ratio) even though it's not used for now, but I'll drop the intermediate counting columns (ref_frags, data_frags).

**Comment by bkieft-usa** [comment]
Should be ready to review again now that the unit tests are all passing. I sent a gchat message about actually implementing the cutoff, let me know if that was your plan or if we should try to get it in upstream of the notebook.

Once you review it you can let me know and I'll do the merge, or you can just merge it in. Maybe hold off on closing this branch until you test it out on Wilkins?

**Review by tharwood3 — APPROVED** [review]
I think you are good to merge in when ready! Keeping the branch open for a bit is a good idea, I'll test the new filter and atlases on a real dataset soon and then I think we are safe to delete it.

**Pull request merged**
